src/app/pages/query-page/query-page.component.html

```diff
@@ -380,7 +414,7 @@ <h3>Something went wrong!</h3>
             </table>
           </mat-tab>
           <mat-tab label="Requested Query">
-            <pre>{{ this.result.query | query }}</pre>
+            <pre>{{ this.result.query | json }}</pre>
```

Comment:
Why use `json` here instead of `query`?

Reply:
Uhm, this line shouldn't have been committed, it's a leftover from a previous version of the frontend. I'll fix it.

Reply:
Done.